### ISSUE TITLE
Module completion statement does not always reach LRS

### ISSUE THREAD

**Issue by kevindoherty30** [adaptlearning/adapt-contrib-xapi#108]
We have seen some intermittent issues with the final module completion statement not being received in an LRS when using xAPI content.

This has proved difficult to recreate but have seen it occurring for a small percentage of users.

This has been when using:

- f/w 5.18.4
- xAPI v 0.8.3

We are looking to update to the latest version of the xAPI extension but I was wondering if there was any appetite for an enhancement to the extension that would monitor for a response from the LRS or resend the statement?

**Comment by oliverfoster** [comment]
Yes please, a send queue is always a good idea for intermittent connections.
